script.js

```diff
@@ -1,6 +1,18 @@
 const toggleButton = document.getElementsByClassName('toggle-button')[0];
 const navBarLinks = document.getElementsByClassName('menu-links')[0];
```

Comment:
- Hey Team, as a result of the missing opening `<li>` tags, the selected class is currently having no effect and the links do not close when clicked by the user. please, kindly fix this issue.

Reply:
This is fixed, the tags previously did not close the menu since there was no listener applied to them, but now it has been fixed, the comment above was not related to the issue